src/main/java/org/dependencytrack/event/kafka/processor/IntegrityAnalysiResultProcessor.java

```diff
@@ -0,0 +1,115 @@
+package org.dependencytrack.event.kafka.processor;
+
+import alpine.common.logging.Logger;
+import alpine.common.metrics.Metrics;
+import com.github.packageurl.MalformedPackageURLException;
+import com.github.packageurl.PackageURL;
+import io.micrometer.core.instrument.Timer;
+import org.apache.kafka.streams.processor.api.Processor;
+import org.apache.kafka.streams.processor.api.Record;
+import org.dependencytrack.model.Component;
+import org.dependencytrack.model.IntegrityAnalysisComponent;
+import org.dependencytrack.persistence.QueryManager;
+import org.hyades.proto.repometaanalysis.v1.HashMatchStatus;
+import org.hyades.proto.repometaanalysis.v1.IntegrityResult;
+
+import javax.jdo.JDODataStoreException;
+import javax.jdo.PersistenceManager;
+import javax.jdo.Query;
+import javax.jdo.Transaction;
+import java.util.Date;
+import java.util.UUID;
+
+public class IntegrityAnalysiResultProcessor implements Processor<String, IntegrityResult, Void, Void> {
```

Comment:
Typo i think in IntegrityAnalysiResultProcessor 
 missing 's' ..IntegrityAnalysisResultProcessor